crates/router/src/connector/zen/transformers.rs

```diff
@@ -0,0 +1,393 @@
+    browser_details: ZenBrowserDetails,
+    #[serde(rename = "type")]
+    payment_type: ZenPaymentTypes,
+    card: ZenCardDetails,
+    descriptor: String,
+    return_verify_url: Option<String>,
+}
+
+#[derive(Debug, Serialize, Eq, PartialEq, frunk::LabelledGeneric)]
+#[serde(rename_all = "camelCase")]
+pub struct ZenBrowserDetails {
+    color_depth: String,
+    java_enabled: bool,
+    lang: String,
+    screen_height: String,
+    screen_width: String,
+    timezone: String,
+    accept_header: String,
+    window_size: String,
+    user_agent: String,
+}
+
+#[derive(Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "snake_case")]
+pub enum ZenPaymentTypes {
+    Onetime,
+}
+
+#[derive(Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct ZenCardDetails {
+    number: Secret<String, pii::CardNumber>,
+    expiry_date: Secret<String>,
+    cvv: Secret<String>,
+}
+
+#[derive(Debug, Serialize, Eq, PartialEq)]
+#[serde(rename_all = "camelCase")]
+pub struct ZenItemObject {
+    name: String,
+    price: String,
+    quantity: u16,
+    line_amount_total: String,
+}
+
+impl TryFrom<&types::PaymentsAuthorizeRouterData> for ZenPaymentsRequest {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(item: &types::PaymentsAuthorizeRouterData) -> Result<Self, Self::Error> {
+        let browser_info = item.request.get_browser_info()?;
+        let order_details = item.request.get_order_details()?;
+
+        let window_size = match (browser_info.screen_height, browser_info.screen_width) {
+            (250, 400) => "01",
+            (390, 400) => "02",
+            (500, 600) => "03",
+            (600, 400) => "04",
+            _ => "05",
+        }
+        .to_string();
+        let browser_details = ZenBrowserDetails {
+            color_depth: browser_info.color_depth.to_string(),
+            java_enabled: browser_info.java_enabled,
+            lang: browser_info.language,
+            screen_height: browser_info.screen_height.to_string(),
+            screen_width: browser_info.screen_width.to_string(),
+            timezone: browser_info.time_zone.to_string(),
+            accept_header: browser_info.accept_header,
+            window_size,
+            user_agent: browser_info.user_agent,
+        };
+        let (payment_specific_data, payment_channel) =
+            match item.request.payment_method_data.clone() {
+                api::PaymentMethodData::Card(ccard) => Ok((
+                    ZenPaymentData {
+                        browser_details,
+                        //Connector Specific for cards
+                        payment_type: ZenPaymentTypes::Onetime,
+                        card: ZenCardDetails {
+                            number: ccard.card_number.clone(),
+                            expiry_date: Secret::new(format!(
+                                "{}{}",
+                                ccard.card_exp_month.clone().expose(),
```

Comment:
Use utils::get_card_expiry_month_year_2_digit_with_delimiter with "" for delimiter